src/main/java/seedu/address/logic/parser/CountryNoteCommandParser.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_COUNTRY;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NOTE;
+
+import seedu.address.logic.commands.CountryNoteCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.country.Country;
+import seedu.address.model.note.Note;
+
+public class CountryNoteCommandParser implements Parser<CountryNoteCommand> {
+
+    @Override
+    public CountryNoteCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_COUNTRY, PREFIX_NOTE);
+
+        if (argMultimap.getValue(PREFIX_COUNTRY).isEmpty() || argMultimap.getValue(PREFIX_NOTE).isEmpty()) {
+            throw new ParseException("invalid message"); // TODO: better messages
```

Comment:
Maybe can add extracting out the messages into a static final string into the `TODO: better messages`, in case we forget

Reply:
Added this as follow-up issue.